NGit.Test/NGit.Api/ApplyCommandTest.cs

```diff
@@ -78,10 +90,10 @@ private ApplyResult Init(string name, bool preExists, bool postExists)
 			}
 			if (postExists)
 			{
-				b = new RawText(ReadFile(name + "_PostImage"));
+				b = new RawText(ReadFile(name + "_PostImage", a != null && a.GetLineDelimiter() == "\r\n"));
```

Comment:
Shouldn't this condition be `(a != null && a.GetLineDelimiter() == "\r\n") || (a == null && m_UseCrlfFiles)`?

i.e. use the same as `a` if it exists, otherwise use `m_UseCrlfFiles`